.vscode/launch.json

```diff
@@ -7,94 +7,251 @@
         {
             "name": "Identity",
+            "configurations": [
+                "run-Identity"
+            ],
             "presentation": {
                 "hidden": false,
                 "group": "cloud",
-                "order": 10
+            },
+            "preLaunchTask": "buildIdentity",
+        },
+        {
+            "name": "Notifications",
+            "configurations": [
+                "run-Notifications"
+            ],
+            "presentation": {
+                "hidden": false,
+                "group": "cloud",
+            },
+            "preLaunchTask": "buildNotifications",
+        },
+        {
+            "name": "SSO",
+            "configurations": [
+                "run-Sso"
+            ],
+            "presentation": {
+                "hidden": false,
+                "group": "cloud",
+            },
+            "preLaunchTask": "buildSso",
+        },
+        {
+            "name": "Admin Self Host",
+            "configurations": [
+                "run-Admin-SelfHost"
+            ],
+            "presentation": {
+                "hidden": false,
+                "group": "self-host",
+            },
+            "preLaunchTask": "buildAdmin",
+        },
+        {
+            "name": "API Self Host",
+            "configurations": [
+                "run-API-SelfHost"
+            ],
+            "presentation": {
+                "hidden": false,
+                "group": "self-host",
+            },
+            "preLaunchTask": "buildAPI",
+        },
+        {
+            "name": "Events Processor Self Host",
+            "configurations": [
+                "run-EventsProcessor-SelfHost"
+            ],
+            "presentation": {
+                "hidden": false,
+                "group": "self-host",
+            },
+            "preLaunchTask": "buildEventsProcessor",
+        },
+        {
+            "name": "Identity Self Host",
+            "configurations": [
+                "run-Identity-SelfHost"
+            ],
+            "presentation": {
+                "hidden": false,
+                "group": "self-host",
+            },
+            "preLaunchTask": "buildIdentity",
+        },
+        {
+            "name": "Notifications Self Host",
+            "configurations": [
+                "run-Notifications-SelfHost"
+            ],
+            "presentation": {
+                "hidden": false,
+                "group": "self-host",
+            },
+            "preLaunchTask": "buildNotifications",
+        },
+        {
+            "name": "SSO Self Host",
+            "configurations": [
+                "run-Sso-SelfHost"
+            ],
+            "presentation": {
+                "hidden": false,
+                "group": "self-host",
+            },
+            "preLaunchTask": "buildSso",
+        },
+    ],
+    "configurations": [
+        // Configurations represent run-only scenarios so that they can be used in multiple compounds
```

Comment:
❓ Invalid to have a comment here right?

Reply:
for json, technically, yes, but vscode is fine with comments in json.